be/src/vec/sink/vmysql_result_writer.cpp

```diff
@@ -715,6 +732,18 @@ Status VMysqlResultWriter::append_block(Block& input_block) {
             }
             break;
         }
+        case TYPE_MAP: {
+            if (type_ptr->is_nullable()) {
+                auto& nested_type =
+                        assert_cast<const DataTypeNullable&>(*type_ptr).get_nested_type(); //for map
+                status = _add_one_column<PrimitiveType::TYPE_MAP, true>(column_ptr, result,
```

Comment:
_add_one_column has been modified by struct, please check this logic.

Reply:
done !